docs/concepts-messaging.md

```diff
@@ -427,6 +427,13 @@ Consumer<byte[]> consumer = pulsarClient.newConsumer()
         .subscribe();
 ```
 
+:::note
+
+Send messages by synchronous API `send` will disable batching, and the message will be sent individually.
```

Comment:
This comment isn't accurate. The batching isn't disabled. The current batch is triggered immediately after sending the message. https://github.com/apache/pulsar/blob/ffff639a1b73a34bbb5115503d4c7783bb2a2770/pulsar-client/src/main/java/org/apache/pulsar/client/impl/TypedMessageBuilderImpl.java#L82-L86

Reply:
Yes I know it, it's just that my statement is not accurate.

How about change it to: 
```txt
Send messages by synchronous API `send` will trigger the batch to be sent immediately, even the batch is not full.
It is for the purpose of reducing the latency of sending messages and preventing blocking of the caller's thread.
```